packages/teams-js/src/internal/communication.ts

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable @typescript-eslint/ban-types */
 /* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable strict-null-checks/all */
```

Comment:
`strictNullChecks` is checked by both the TS compiler and eslint. Turning off the eslint checking of `strictNullChecks` here does not put us in a worse position because the compiler is still checking for it in this file. It just means that I don't have to suppress the violations BOTH for the linter and the compiler.

Eventually I will just remove the lint rule detection because the compiler detection is sufficient. But not yet.